app/components/Settings/UserConnections/filedrop.jsx

```diff
@@ -0,0 +1,175 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import {SAMPLE_DBS} from '../../../constants/constants';
+
+export default class Filedrop extends Component {
+    static propTypes = {
+        settings: PropTypes.object,
+        connection: PropTypes.object,
+        updateConnection: PropTypes.func,
+        sampleCredentialsStyle : PropTypes.object
+    }
+
+    /**
+     * Filedrop is an input component where users can type an URL or drop a file
+     * 
```

Comment:
Code looks clean and easy to understand.  Do you think this component is worth having a Jest test for or is this overkill?

Reply:
I've just added the tests